tests/v2/actions/fleet/fleet.go

```diff
@@ -3,24 +3,36 @@ package fleet
 import (
 	"errors"
 	"strings"
-	"testing"
 	"time"
 
 	"github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
 	"github.com/rancher/shepherd/clients/rancher"
 	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+
 	extensionsfleet "github.com/rancher/shepherd/extensions/fleet"
 	"github.com/rancher/shepherd/extensions/workloads/pods"
 	"github.com/rancher/shepherd/pkg/config"
 	"github.com/sirupsen/logrus"
-	"github.com/stretchr/testify/require"
 	appsv1 "k8s.io/api/apps/v1"
 	kwait "k8s.io/apimachinery/pkg/util/wait"
 )
 
 // The json/yaml config key for the corral package to be build ..
 const (
 	gitRepoConfigConfigurationFileKey = "gitRepo"
+
+	FleetName          = "fleet"
+	LocalName          = "local"
+	HarvesterName      = "harvester"
+	ExampleRepo        = "https://github.com/rancher/fleet-examples"
+	BranchName         = "master"
```

Comment:
might be best to maintain these two in the tests itself as they are subject to change  depending upon the test. lmk what you think.

Reply:
for pretty much all of the tests I'm using, these will be the repo and branch. So I think it makes sense to keep it

Reply:
ok then we can maintain these in fleet.go. thanks!